pkg/kubeclient/kubelet_client.go

```diff
@@ -166,6 +169,9 @@ func (client *KubeletClient) GetMachineInfo(host string) (*cadvisorapi.MachineIn
 		if entryPresent {
```

Comment:
Shall we add the extra condition check here? The thing is we are using one cache object for two different information, I am not sure if summary is nil, we should count as cache used.